packages/bahmni-widgets/src/search/commonSearch/__tests__/utils.test.ts

```
@@ -3,7 +3,7 @@ import {
   formatGender,
   resolveComboBoxItems,
 } from '@bahmni/services';
-import { LookupOption, TextInput } from '../models';
+import { LookupOption, CriterionConfig, CriterionRow, TextInput } from '../models';
```

ESLint

[error] Replace `·LookupOption,·CriterionConfig,·CriterionRow,·TextInput·` with `⏎··LookupOption,⏎··CriterionConfig,⏎··CriterionRow,⏎··TextInput,⏎`

(prettier/prettier)